packages/grid/x-grid/src/XGrid.test.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+// @ts-ignore
```

Comment:
why do we need this one here?

Reply:
No idea

Reply:
Ok, I have looked at why. The test utils haven't .d.ts type definitions nor written in .ts.

I don't fully understand how this work https://github.com/mui-org/material-ui/blob/980e84b9a2454b3ba72c7562fe0655c6b72f853c/packages/material-ui-lab/src/TabPanel/TabPanel.test.tsx#L3, it seems to be thanks to JSDOC.